### PR TITLE
Adding wlcg conversion

### PR DESCRIPTION
**Pull request by djw8605**
Add WLCG Conversions

### CODE REVIEW

**Comment by djw8605** [comment]
@jthiltges could you take a look at this.

File: Collectors/DetailedCollector.py [hunk 1]
```diff
@@ -144,13 +147,22 @@ def addRecord(sid, userID, fileClose, timestamp, addr):
     rec['read'] = fileClose.read
     rec['readv'] = fileClose.readv
     rec['write'] = fileClose.write
+
```
Comment (jthiltges) [inline Collectors/DetailedCollector.py]: Trailing whitespace here and on line 160.

File: Collectors/wlcg_converter.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import uuid
+
+
+def Convert(source_record):
+    """
+    Convert to the WLCG format documented here:
+    https://twiki.cern.ch/twiki/bin/view/Main/GenericFileMonitoring
+    """
+    to_return = {}
+    to_return["site_name"] = source_record.get('site', "")
```
Comment (jthiltges) [inline Collectors/wlcg_converter.py]: Would be nice to keep the quotes consistent, double or single

Comment (djw8605) [reply Collectors/wlcg_converter.py]: Fixed!

File: Collectors/DetailedCollector.py (hunk 1 above)
Comment (djw8605) [reply Collectors/DetailedCollector.py]: Ugh, at my IDE

**Pull request merged**